clouddriver-ecs/src/main/java/com/netflix/spinnakerclouddriver/aws/ecs/EcsCloudProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnakerclouddriver.aws.ecs;
+
+import org.springframework.stereotype.Component;
+import com.netflix.spinnaker.clouddriver.core.CloudProvider;
+
+import java.lang.annotation.Annotation;
+
+//@Component
```

Comment:
@tomaslin So you'd rather have the importing of EcsConfiguration removed, as well as the project dependency in clouddriver-web.gradle?  Or did I misunderstand what you wrote?